src/core/core.scale.js

```diff
@@ -341,7 +319,7 @@ module.exports = Element.extend({
 
 		// Get the width of each grid by calculating the difference
 		// between x offsets between 0 and 1.
-		var tickFont = parseFontOptions(tickOpts);
+		var tickFont = helpers.options.parseFontOptions(tickOpts, globalDefaults);
```

Comment:
Would it make sense to capture `helpers.options.parseFontOptions` into a variable to help minification?